bridgetown-core/lib/bridgetown-core/configurations/lit.rb

```diff
@@ -59,7 +59,7 @@
       // Read https://www.bridgetownrb.com/docs/components/lit#sidecar-css-files for documentation.
       /*
       postCssPluginConfig: {
-        filter: /(?:index|\.global)\.css$/,
+        filter: /(?:index|\\.global)\\.css$/,
```

Comment:
So I noticed that this actually changes the output, which I don't think we want.

```diff
- filter: /(?:index|.global).css$/,
+ filter: /(?:index|\.global)\.css$/,
```

I'm no regex expert but given this, I don't think we want this change or the one on line 41. Wdyt?